jobconfig.toml

```diff
@@ -16,8 +16,10 @@
 ## "# Total Positions" should always be changed to "Total Positions", no additional spacing.
 ## Best of luck editing!
 
+## Edit as of fork "Job-Limit-Increases": All jobs that I considered reasonable to have a limit increase have been increased.
+
 [AI]
-"Playtime Requirements" = 2400
+"Playtime Requirements" = 1200
```

Comment:
Don't include changes from other independent PRs like this- instead let git handle merging the two together.

Reply:
Next commit reverts this change.